src/gatherers/service-worker.js

```diff
@@ -41,8 +43,18 @@ class ServiceWorker extends Gather {
     });
   }
 
+  static getOrigin(url) {
+    if (typeof window.URL === 'function') {
+      return new window.URL(url).origin;
+    }
+
+    const parsedURL = require('url').parse(url);
```

Comment:
`url` is shimmed by browserify. Will using it already work in the extension and can just drop the `window.URL` conditional/return above?